runtime/src/main/java/org/corfudb/runtime/LiteRoutingQueueListener.java

```diff
@@ -183,7 +178,35 @@ public CorfuStoreMetadata.Timestamp performFullSync() {
         return ts;
     }
 
+    private int processBatch(ReplicationType currentReplicationType, List<Queue.RoutingTableEntryMsg> batch) {
+        int entriesProcessed = 0;
+        switch (currentReplicationType) {
```

Comment:
this switch statement and `processBatch` look similar. Can they be consolidated?